### PR TITLE
MRG: Allow del_proj to delete all projs

### PR DESCRIPTION
**Pull request by Eric89GXL**
Up until now I've been doing `raw.add_proj([], remove_existing=True)`, which is silly. Now we can do `raw.del_proj()`.

### CODE REVIEW

File: mne/io/proj.py [hunk 1]
```diff
@@ -177,27 +177,30 @@ def apply_proj(self):
...
         Note: The projection vector can only be removed if it is inactive
               (has not been applied to the data).
 
         Parameters
         ----------
-        idx : int
-            Index of the projector to remove.
+        idx : int | str
+            Index of the projector to remove. Can also be "all" to remove
+            all projectors.
```
Comment (agramfort) [inline mne/io/proj.py]: Say that it defaults to 'all'. Then +1 for merge.

**Comment by Eric89GXL** [comment]
@agramfort done

File: mne/io/proj.py [hunk 2]
```diff
@@ -177,27 +177,30 @@ def apply_proj(self):
...
         """Remove SSP projection vector.
 
         Note: The projection vector can only be removed if it is inactive
               (has not been applied to the data).
 
         Parameters
         ----------
-        idx : int
-            Index of the projector to remove.
+        idx : int | str
```
Comment (teonbrooks) [inline mne/io/proj.py]: int | list of int | str

**Comment by Eric89GXL** [comment]
Thanks @teonbrooks, fixed

**Comment by teonbrooks** [comment]
LGTM

**Pull request merged**